float64 gripper
---
int64 error_code
